client/state/themes/actions.js

```diff
@@ -75,26 +75,34 @@ export function incrementThemesPage( site ) {
 	};
 }
 
-export function fetchCurrentTheme( siteId ) {
+/**
+	* This action queries wpcom endpoint for active theme for site.
```

Comment:
Indentation -- there should be a space, not a tab before the `*` so it aligns with the first `*` in `/**`.